gtest/src/manager.rs

```diff
@@ -223,6 +223,7 @@ pub(crate) struct ExtManager {
     pub(crate) wait_list: BTreeMap<(ProgramId, MessageId), StoredDispatch>,
     pub(crate) wait_init_list: BTreeMap<ProgramId, Vec<MessageId>>,
     pub(crate) gas_limits: BTreeMap<MessageId, Option<u64>>,
+    pub(crate) delayed_dispatches: Vec<Dispatch>,
```

Comment:
Why do you need to process all of the delayed dispatches? It should be only if we already spent the expected for dispatch block number